### PR TITLE
Bump up the formidable package version.

### PR DESCRIPTION
**Pull request by arikt-ms**
# [Bump up the "formidable" package version to address a vulnerability issue](./link-to-the-work-item)

For more information about how to contribute to this repo, visit this [page](https://github.com/microsoft/FluidFramework/blob/main/CONTRIBUTING.md)

## Description

The currently used version of ["formidable" package](https://www.npmjs.com/package/formidable/v/3.2.4) has a [vulnerability bug](https://github.com/node-formidable/formidable/issues/856) which has been fixed in the latest version (3.2.4). Bumping up the version to the latest to address the vulnerability issue.

## Steps to Reproduce Bug and Validate Solution

> Only applicable if the work is to address a bug. Please remove this section if the work is for a feature or story
> Provide details on the environment the bug is found, and detailed steps to recreate the bug.
> This should be detailed enough for a team member to confirm that the bug no longer occurs

## PR Checklist

> Use the check-list below to ensure your branch is ready for PR. If the item is not applicable, leave it blank.

-   [ ] I have updated the documentation accordingly.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] My code follows the code style of this project.
-   [x] I ran the lint checks which produced no new errors nor warnings for my changes.
-   [x] I have checked to ensure there aren't other open Pull Requests for the same update/change.

## Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

> If this introduces a breaking change, please describe the impact and migration path for existing applications below.

## Testing

> -   Instructions for testing and validation of your code so the reviewer can follow those steps and validate code works as expected

## Any relevant logs or outputs

> -   Use this section to provide either screenshots or output of logs as code snippets

## Other information or known dependencies

> -   Any other information or known dependencies that is important to this PR.
> -   TODO that are to be done after this PR.

### CODE REVIEW

**Review by hedasilv — APPROVED** [review]
The change looks good to me! I would just recommend waiting for @znewton 's review as well since it seems `formidable` is used in the RestLess Server implementation, added by Zach.

**Review by znewton — APPROVED** [review]
It looks like `formidable@3.0.0` removed the "multiples" option, which we use. However, I think they removed it in a way that works for us. Otherwise, looks like no breaking changes. Would you mind removing `{ multiples: true }` from line 103 of server/routerlicious/packages/services-shared/src/restLessServer.ts? Also, please bump `@types/formidable` to `2.0.5`

Thanks, Arik!

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,14991 @@
 {
   "name": "root",
   "version": "0.14.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (znewton) [inline package-lock.json:4]: You have the wrong version of NPM installed. lockfileVersion should be `1`. Please use Node v14.19.0 and NPM v6

**Review by znewton — CHANGES_REQUESTED** [review]

**Comment by znewton** [comment]
Closing in favor of #10451 , thanks Arik for working with me on this!

**Pull request closed without merging**